python_ta/reporters/node_printers.py

```diff
@@ -614,6 +614,31 @@ def render_pep8_errors_e306(msg, _node, source_lines=None):
     yield from render_context(msg.line, msg.line + 2, source_lines)
 
 
+def render_missing_return_statement(msg, _node, source_lines=None):
+    """
+    Render a missing return statements message
+    """
+    # add function name to reported message
+    msg.msg += f" `{_node.name}`"
+
+    yield from render_context(msg.line - 1, msg.line + 1, source_lines)
+
+    # calculate indentation for the insertion point
+    body = source_lines[msg.line - 1]
+    indentation = len(body) - len(body.lstrip())
+    insertion_text = body[:indentation] + "# INSERT RETURN STATEMENT HERE"
```

Comment:
I'm thinking about this a bit more and I think we need to give a bit more flexibility in the suggested fix. E.g. "INSERT RETURN STATEMENT HERE (OR BELOW)" in case students want to write a final return statement at the end of the function body. I think it should be possible to detect whether we're at the end of the function body or not, and if so add the extra text to the message